pkg/iostreams/color.go

```diff
@@ -43,13 +43,14 @@ var (
 
 // NewColorScheme initializes color logic based on provided terminal capabilities.
 // Logic dealing with terminal theme detected, such as whether color is enabled, 8-bit color supported, true color supported,
-// and terminal theme detected.
-func NewColorScheme(enabled, is256enabled, trueColor, accessibleColors bool, theme string) *ColorScheme {
+// labels are colored, and terminal theme detected.
+func NewColorScheme(enabled, is256enabled, trueColor, accessibleColors, colorLabels bool, theme string) *ColorScheme {
 	return &ColorScheme{
 		enabled:          enabled,
 		is256enabled:     is256enabled,
 		hasTrueColor:     trueColor,
 		accessibleColors: accessibleColors,
+		colorLabels:      colorLabels,
 		theme:            theme,
 	}
 }
```

Comment:
@cli/code-reviewers : What are your thoughts about refactoring these booleans into custom types in order to make calling code easier to understand?

Just going to say I don't really like this as increasing number of booleans makes calling code difficult to read.  I was waiting for this PR to discuss it.

Reply:
Discussing this internally, @williammartin brought up a good point that an alternative to custom types is simply exporting the fields and allowing the calling code to pass the necessary values.

This does make testing easier as the default values generally suffice, though it means that future changes will need to be diligent in all the places this type is used.